hopbox/hopbox_app/templates/edit_account.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Update Account</title>
+    <!-- {% load static %} -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-white  bg-white mb-5" style="border-bottom: 3px solid #91B957;">
+        <div class="container-fluid d-flex justify-content-evenly">
+            <a class="navbar-brand ms-2 fw-bold active" href="/userdashboard">
```

Comment:
we should think about changing the urls to follow something more RESTful -- i.e. '/user/dashboard,' '/user/login,' etc,